package.json

```diff
@@ -43,7 +44,7 @@
     "postinstall": "npx husky install",
     "lint:check": "solhint 'contracts/**/*.sol' 'interfaces/**/*.sol' && prettier --check './**'",
     "lint:fix": "sort-package-json && prettier --write './**' && solhint --fix 'contracts/**/*.sol' 'interfaces/**/*.sol'",
-    "prepublishOnly": "hardhat clean && PUBLISHING_NPM=true hardhat compile && pinst --disable",
+    "prepublishOnly": "hardhat clean && PUBLISHING_NPM=true hardhat compile && yarn transpile && pinst --disable",
```

Comment:
Can we now remove all `TS_NODE_SKIP_IGNORE=true` from the other commands and the github workflow?

Reply:
No we can't, sadly. Executing ANYTHING means loading `hardhat.config.ts`, which in turn means loading `${mean-package}/deploy`, that needs `TS_NODE_SKIP_IGNORE`.